src/_cost-storage.ts

```
@@ -0,0 +1,204 @@
+ * Appends new cost entry to existing data or creates new file
+ * @param sessionId - Session ID
+ * @param totalCostUsd - Total cost in USD from Claude Code
+ * @param timestamp - ISO timestamp string (defaults to current time)
+ * @returns Result indicating success or failure
+ */
+export async function saveCostData(
+	sessionId: string,
+	totalCostUsd: number,
+	timestamp?: string,
+): Promise<Result<void, Error>> {
+	return Result.pipe(
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)